extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/DefaultStatelessSessionProducer.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.hibernate.orm.runtime;
+
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.persistence.EntityManagerFactory;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.StatelessSession;
+
+public class DefaultStatelessSessionProducer {
+
+    @Inject
+    EntityManagerFactory em;
```

Comment:
This would probably fail if a custom `EntityManagerFactory` producer is present (`DefaultEntityManagerFactoryProducer` is only registered if such producer does not exist).

Reply:
Why would it fail?

Reply:
Oh, sorry. It would fail if a user-provided producer declares some qualifier... such as `@Documents` in:
```java
@Produces @PersistenceUnit(unitName="documents") @Documents EntityManagerFactory factory;
```
But it's true that there is no reason to declare a producer like this if only one PU is supported...